packages/app/features/accounts/styled-input.tsx

```diff
@@ -0,0 +1,50 @@
+import { TextInput as rTextInput } from 'react-native'
+import { Ref, useState } from 'react'
+import { TextInput, View } from 'app/design-system'
+import Icon from '@expo/vector-icons/MaterialCommunityIcons'
+
+interface StyledInputProps {
+  placeholder: string
+  autofocus?: boolean
+  value?: string
+  onChangeText?: (s: string) => void
+  className?: string
+  valid?: boolean
+  inputRef?: Ref<rTextInput>
+}
+
+export function StyledInput({
```

Comment:
this name says nothing about what is the responsibility of the component

Reply:
Changed to StyledTextInput